arch/arm/src/tlsr82/tc32/tc32_exception.S

```diff
@@ -0,0 +1,234 @@
+/****************************************************************************
+ * arch/arm/src/tlsr82xx/tc32/tc32_exception.S
```

Comment:
seems like all the code is commented out

Reply:
Yes, the common exception implementation is in cstartup_flash.S, __irq, line 411.

Reply:
If so, should we remove this file.

Reply:
Yes and Done.